src/test/java/net/openhft/chronicle/wire/MessageHistoryTest.java

```diff
@@ -77,8 +77,8 @@ public void checkSerialiseBytes() {
         history2.readMarshallable(bw);
         assertEquals("VanillaMessageHistory { " +
                 "sources: [1=0xff,2=0xfff], " +
-                "timings: [1000000000000000000,1000000000000010000,120962203520100], " +
-                "addSourceDetails=false }", history2.toString());
+                "timings: [1000000000000000000,1000000000000010000,120962203520100,120962203520100], " +
```

Comment:
why different?

Reply:
Added addSourceDetails(true); to SetTimeMessageHistory for the benefit of other tests